core/src/main/java/org/apache/iceberg/puffin/StandardBlobTypes.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.puffin;
+
+public final class StandardBlobTypes {
+  private StandardBlobTypes() {
+  }
+
+  /**
+   * 8-bytes integer stored little-endian and representing number of distinct values
+   */
+  public static final String NDV_LONG_LITTLE_ENDIAN = "ndv-long-little-endian";
```

Comment:
This was removed from the spec, so we should also remove it from this commit.

Reply:
of course, removed!